arrebol/worker/whitelist/whitelist.go

```diff
@@ -0,0 +1,60 @@
+package whitelist
+
+import (
+	"bufio"
+	"log"
+	"os"
+)
+
+const (
+	WhiteListPath = "WHITE_LIST_PATH"
+)
+
+type WhiteList interface {
+	Contains(id string) (bool)
+}
+
+type FileWhiteList struct {
```

Comment:
Didn't like this name. Let's make things simple, do we really need the WhiteLIst interface? We should call this struct WhiteList, and get rid of the interface

Reply:
I would not remove the interface. It increases modularization and facilitates possible new white list implementations. What do you think of renaming it to DefaultWhiteList?

Reply:
I think we won't have another impl of the whiteList. Besides, if we need another implementation we add the interface. I think that my first suggestion is better for now..

Reply:
Done.